modules/test/test_tell.py

```diff
@@ -35,10 +35,202 @@ def test_messageAlert(self):
+        tell.nick_aliases = []
+
+        aliases = ['tester', 'testing', 'testmaster']
+
+        for alias in aliases:
+            self.create_alias(alias, self.input)
+
+        aliases.insert(0,'Testsworth')
+
+        aligroup = tell.aliasGroupFor('Testsworth')
+        self.assertTrue(aliases == aligroup)
+
+    def test_aliasGroupFor_singleton(self):
+        self.input.nick = 'Testsworth'
+        tell.nick_aliases = []
+        aliases = ['Testsworth']
+
+        aligroup = tell.aliasGroupFor('Testsworth')
+        self.assertTrue(aliases == aligroup)
+
+    def test_aliasPairMerge(self):
+        self.input.nick = 'Testsworth'
+        tell.nick_aliases = []
+        aliases = ['tester', 'testing', 'testmaster']
+        for alias in aliases:
+            self.create_alias(alias, self.input)
+
+        self.input2.nick = 'Happy'
+        aliases2 = ['joyful', 'ecstatic', 'euphoric', 'blissful']
+        for alias in aliases2:
+            self.create_alias(alias, self.input2)
+
+        tell.aliasPairMerge(self.phenny, 'Testsworth', 'Happy')
+
+        aliases.insert(0,'Testsworth')
+        aliases2.insert(0,'Happy')
+        joined = aliases + aliases2
+
+        self.assertTrue(joined in tell.nick_aliases)
+
+    def test_alias_noadded(self):
+        self.input.nick = 'Testsworth'
+        self.input.group = lambda x: ['alias', 'add', None][x]
+
+        tell.alias(self.phenny, self.input)
+        self.phenny.reply.assert_called_once_with("Usage: .alias add <nick>")
+
+    def test_alias_addself(self):
+        self.input.nick = 'Testsworth'
+        self.input.group = lambda x: ['alias', 'add', 'Testsworth'][x]
+
+        tell.alias(self.phenny, self.input)
+        self.phenny.reply.assert_called_once_with("I don't think that will be necessary.")
+
+    def test_alias_alreadypaired(self):
+        self.input.nick = 'Testsworth'
+        tell.nick_aliases = [['Testsworth', 'tests']]
+
+        self.input.group = lambda x: ['alias', 'add', 'tests'][x]
+
+        tell.alias(self.phenny, self.input)
+        self.phenny.reply.assert_called_once_with("You and tests are already paired.")
+
+    def test_alias_confirmreq(self):
+        self.input.nick = 'Testsworth'
+        tell.nick_aliases = []
+        tell.nick_pairs.append(['tests', 'Testsworth'])
+
+        self.input.group = lambda x: ['alias', 'add', 'tests'][x]
+
+        tell.alias(self.phenny, self.input)
+        self.phenny.reply.assert_called_once_with("Confirmed alias request with tests. Your current aliases are: Testsworth, tests.")
+
+    def test_alias_alreadysent(self):
+        self.input.nick = 'Testsworth'
+        tell.nick_aliases = []
+        tell.nick_pairs.append(['Testsworth', 'tests'])
+
+        self.input.group = lambda x: ['alias', 'add', 'tests'][x]
+
+        tell.alias(self.phenny, self.input)
+        self.phenny.reply.assert_called_once_with("Alias request already exists. Switch your nick to tests and call \".alias add Testsworth\" to confirm.")
```

Comment:
Some other similar instances in this file.
```suggestion
        self.phenny.reply.assert_called_once_with('Alias request already exists. Switch your nick to tests and call ".alias add Testsworth" to confirm.')
```